src/opera/test/pge/base_pge_test/scratch/testMetFile.met

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
It looks like this file is only meant to be a test file that got checked in by accident. The `test_metfile.testMetFile` function needs to be updated to check for and remove this file if present at the end of the test. The check-and-delete should always occur regardless of whether the test passes or fails:


```python
met_file = MetFileTestCase.TEST_MET_FILE
try:
   <tests with assert statements>
finally:
    if os.path.exists(met_file):
        os.unlink(met_file)
```

As-is, `test_metfile.py` is failing in the CI system since this file can exist from a previous test run and fails when a new test run tries to overwrite due to permission issues.

Reply:
This file is not being used anymore, and it was deleted from git.